src/app/clusters/scenes/scenes.cpp

```diff
@@ -501,7 +501,8 @@ bool emberAfScenesClusterGetSceneMembershipCallback(chip::app::Command * command
         VerifyOrExit((writer = commandObj->GetCommandDataElementTLVWriter()) != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
         SuccessOrExit(err = writer->Put(TLV::ContextTag(0), status));
         SuccessOrExit(err = writer->Put(TLV::ContextTag(1),
-                                        (EMBER_AF_PLUGIN_SCENES_TABLE_SIZE - emberAfPluginScenesServerNumSceneEntriesInUse())));
+                                        static_cast<int32_t>(EMBER_AF_PLUGIN_SCENES_TABLE_SIZE -
```

Comment:
This should be an unsigned int, I would think.  Per spec this is a `uint8`.

Reply:
Fixed